### ISSUE TITLE
Modifiers are not supported, application required

### ISSUE THREAD

**Issue by xiantuan** [addam/Export-Paper-Model-from-Blender#82]
For example：mirror

**Comment by addam** [comment]
This is quite an easy task, there's just one important catch. Features like 'Create UVMap' and 'Bake Texture' operate directly on mesh data, so these would have to be disabled if modifiers are used.

Are there any modifiers apart from Mirror that should be supported? Armature and Boolean come into mind.

**Comment by xiantuan** [comment]
Yes! and exports need to be merged in advance, can you change the selected objects to

**Comment by addam** [comment]
That's a very different request. I also think trying to export several objects at once is the reason of the _other_ issue you reported just now.

The script will rather throw an error if you try to export several objects at once. I'm afraid that the Bake Texture feature is not stable enough.

**Comment by xiantuan** [comment]
Export after Ctrl+J
